portfolio/src/main/webapp/script.js

```diff
@@ -251,3 +252,28 @@ function drawScoreChart() {
 function drawMinChart() {
   drawChart(false);
 }
+
+var markers = [];
+var map;
+
+function createMap() {
+  map = new google.maps.Map(document.getElementById('map-div'), {
+    zoom: 1.8,
+    center: {lat: 30, lng: 0}
+  });
+
+  for (var i = 0; i < locations.length; i++) {
+    addMarkerWithTimeout(locations[i], i * 200);
+  }
+}
+
+function addMarkerWithTimeout(position, timeout) {
+  window.setTimeout(function() {
+    markers.push(new google.maps.Marker({
+      position: new google.maps.LatLng(position[1], position[2]),
+      map: map,
+      animation: google.maps.Animation.DROP,
+      title: position[0]
+    }));
+  }, timeout);
+}
```

Comment:
Empty line at the end of the file?

Reply:
The empty line is there but the github view removes it. If there is no empty line, then a little red circle appears